### DISCUSSION TITLE
Adding Localization should migrate fields to defaultLocale instead of dropping them

### DISCUSSION THREAD

**Discussion by janus-reith** [payloadcms/payload#5660] [Feature Requests & Ideas]
Currrently, when adding localization, fields that now don't match the schema anymore will be dropped (Although there is a warning and it asks for confirmation).

I think it could highly increase DX to offer migrating the current values to the specifed default locale.
Not sure how complicated it is too hook into the current migration logic though.

**Comment by chaotic-justice** [comment]
need this too. currenlty using mongodb, after adding a locale i can fetch the original data by passing in `locale: 'all'`. switching to `locale: 'zh'` fails to default back to the original. the migrations doc is not clear on how to achieve this - migrating richtext data from locale-all to locale-others

**Comment by alexfacciorusso** [comment]
Absolutely needed feature. Migrations should automatically use the default locale and copy that data over instead of dropping it

**Comment by notflip** [comment]
Is there any news on this? I have to localise a field that has a lot of input, for a live website and the client is going to lose all that input?
